spec/system/admin_user_fields_spec.rb

```diff
@@ -40,7 +40,7 @@
 
     user_fields_page.choose_requirement("optional")
 
-    expect(form).to have_field(editable_label, checked: false, disabled: false)
+    expect(form).to have_field(editable_label, checked: true, disabled: false)
```

Comment:
This change in behaviour is really getting rid of an old artefact, where moving away from "for all users" unchecked the box, but the simplest and least surprising thing is to not touch it.